Broken stat visualization for high base stat pokemon

Pokemon with a high enough base stat break the stat bars and cause a complete meltdown in rendering the pokemon summary. The stat-bar bounds are set incorrectly.

Output for a working pokemon using `get pokemon porygon -d`

```
porygon (ID: 137)
──────────────────────────────
Species:   porygon
Height:    80 cm
Weight:    36.50 kg
Types:     normal
Abilities: trace, download, analytic

Stats:
  special-defense  75 [███████   ]
  speed            40 [████      ]
  hp               65 [██████    ]
  attack           60 [██████    ]
  defense          70 [███████   ]
  special-attack   85 [████████  ]

Moves:
  - headbutt
  - tackle
  - take-down
  - double-edge
  - ice-beam
  ...
```

Output for a broken pokemon using `get pokemon mewtwo -d`

```
mewtwo (ID: 150)
──────────────────────────────
Species:   mewtwo
Height:    200 cm
Weight:    122.00 kg
Types:     psychic
Abilities: pressure, unnerve

Stats:
panic: strings: negative Repeat count

goroutine 1 [running]:
strings.Repeat({0x7ff6bc1dfb88?, 0x3?}, 0x7ff6bbeb2f8f?)
        C:/Program Files/Go/src/strings/strings.go:628 +0x585
pokecli/format.StatBar(0x7ff6bc1e34a0?)
        C:/Users/user/Documents/goLang/pokecli/format/transformations.go:30 +0x76
pokecli/format.PrintDetailedPokemonSummary({{0xc000489ea0, 0x6}, 0x96, 0x14, 0x4c4, {0xc000026610, 0x1, 0x1}, {0xc00007caa0, 0x2, ...}, ...}, ...)
        C:/Users/user/Documents/goLang/pokecli/format/render.go:29 +0x413
pokecli/commands.GetPokemon({0xc0000a2030?, 0x2ddc3a60108?, 0x2000000000013?})
        C:/Users/user/Documents/goLang/pokecli/commands/get.go:44 +0x36a
pokecli/controller.Route({0xc0000a2000?, 0x70?, 0x0?})
        C:/Users/user/Documents/goLang/pokecli/controller/router.go:15 +0xb9
main.main()
        C:/Users/user/Documents/goLang/pokecli/main.go:17 +0x74
exit status 2
```

--- a/format/render.go
+++ b/format/render.go
@@ -6,6 +6,15 @@ import (
 	"strings"
 )
 
+var StatOrder = []string{
+	"hp",
+	"attack",
+	"defense",
+	"special-attack",
+	"special-defense",
+	"speed",
+}
+
 func PrintPokemonSummary(summary api.PokemonSummary) {
 	fmt.Printf("%s (ID: %d)\n", summary.Name, summary.ID)
 	fmt.Println("==============================")
@@ -25,7 +34,8 @@ func PrintDetailedPokemonSummary(summary api.PokemonSummary, fullMoveset bool) {
 	fmt.Printf("Abilities: %s\n", AbilitiesDisplay(summary.Abilities))
 
 	fmt.Println("\nStats:")
-	for stat, value := range summary.Stats {
+	for _, stat := range StatOrder {
+		value := summary.Stats[stat]
 		bar := StatBar(value)
 		fmt.Printf("  %-15s %3d %s\n", stat, value, bar)
 	}

--- a/format/transformations.go
+++ b/format/transformations.go
@@ -26,8 +26,12 @@ func HeightToString(height int) string {
 }
 
 func StatBar(value int) string {
-	barLength := value / 10
-	return "[" + strings.Repeat("█", barLength) + strings.Repeat(" ", 10-barLength) + "]"
+	const maxStatValue = 255
+	const maxBarLength = 25
+
+	normalizedBarLength := int(float64(value) / float64(maxStatValue) * float64(maxBarLength))
+
+	return "[" + strings.Repeat("█", normalizedBarLength) + strings.Repeat(" ", maxBarLength-normalizedBarLength) + "]"
 }
 
 func TypesDisplay(types []string) string {

